main/res/layout/fragment_edit_note.xml

```diff
@@ -20,7 +20,7 @@
             android:importantForAutofill="no"
             android:inputType="textMultiLine"
             android:minLines="8"
-            android:textSize="@dimen/textSize_detailsPrimary" />
+            android:textSize="@dimen/textSize_textinputPrimary" />
```

Comment:
Wouldn't it be enough to just remove the definition completely and use the style value?

Reply:
If it is possible it would indeed avoid future searching (like I did before).

Reply:
Updated the PR accordingly